content/300-accelerate/250-connection-pooling.mdx

```diff
@@ -40,7 +40,7 @@ To change the size of the connection pool:
 
 ### Configuring the connection pool timeout
 
-The connection pool timeout is the duration, measured in seconds, during which the query engine must process a specific query; failing to do so within this time frame results in an exception being thrown, and the system proceeds to the next query in the queue.
+The connection pool timeout is the maximum number of seconds to wait for a new connection to be opened. If the connection is not opened within this time frame, an exception is thrown. A value of 0 means there is no timeout.
```

Comment:
> The connection pool timeout is the maximum number of seconds to wait for a new connection to be opened.

A connection from where, to where? I think that's the critical information missing here.

Reply:
```suggestion
The connection pool timeout is the maximum number of seconds to wait for a new connection to be opened. If the connection is not opened from the pool to the database within this time frame, an exception is thrown. A value of 0 means there is no timeout.
```

Reply:
@jharrell is this better?

Reply:
Yeah that's in the right direction. I think what I'm trying to get at is that a user should understand where the problem is going wrong. Is it between Prisma Client and Accelerate? Accelerate and the DB? 

From your update it seems we mean the latter. So I'd tweak your copy to something like this 
```suggestion
The connection pool timeout is the maximum number of seconds that Accelerate's internal connection pool waits for a new connection to your database to succeed and open. If the connection is not opened within this time frame, an exception is thrown. A value of 0 means there is no timeout.
```

Something like that, verbatim not needed just more clear to an end user.